.github/workflows/workflow-docker.yaml

```diff
@@ -28,6 +28,7 @@ on:
   push:
     branches:
       - master
+      - dev-mjt-chemsearchindex2
```

Comment:
Can you revert this part of the PR? (or apply your changes form this PR to the other PR, with this file excluded?)